hoomd/mpcd/collide.py

```diff
@@ -162,17 +164,26 @@ class CollisionMethod(Operation):
 
         Number of integration steps between collisions.
         `Read more... <hoomd.mpcd.collide.CollisionMethod.period>`
+
+    .. py:attribute:: kT
+
+        Temperature of the thermostat.
+        `Read more... <hoomd.mpcd.collide.CollisionMethod.kT>`
     """
     )
 
-    def __init__(self, period, embedded_particles=None):
+    def __init__(self, period, kT=None, embedded_particles=None):
```

Comment:
To make this API change non-breaking, the new argument needs to come at the end:
```suggestion
    def __init__(self, period, embedded_particles=None, kT=None):
```